pkg/apis/types/v1beta1/federatedworkspacerole_types.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2020 KubeSphere Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	rbacv1 "k8s.io/api/rbac/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	ResourcePluralFederatedWorkspaceRole   = "federatedworkspaceroles"
+	ResourceSingularFederatedWorkspaceRole = "federatedworkspacerole"
+	FederatedWorkspaceRoleKind             = "FederatedWorkspaceRole"
+)
+
+// +genclient
```

Comment:
`+genclient` could be deleted when using Contoller-runtime. I don't think the auto-genated clients were used anywhere.

Reply:
done